security-start/src/index.js

```diff
@@ -56,26 +56,7 @@ import { getFirebaseConfig } from "./firebase-config.js";
 
 // Signs-in Friendly Chat.
 async function signIn() {
-  // Sign in Firebase using popup auth and Google as the identity provider.
-  var provider = new GoogleAuthProvider();
-  await signInWithPopup(getAuth(), provider)
-    .then(function (userCredential) {
-      // User successfully signed in and is not enrolled with a second factor.
-    })
-    .catch(function (error) {
-      if (error.code == "auth/multi-factor-auth-required") {
-        alert(`MFA is required. ${error}`);
-
-        // TODO 2: Remove the alert above and uncomment the following code block
```

Comment:
I think you only need to remove this part, not the full signIn() function

Reply:
Oh! Rationale behind this is so devs can play around with sign in with google before the mfa portion is added, but I'll defer to you!

Reply:
+1